DQM/DTMonitorModule/python/dt_dqm_sourceclient_common_cff.py

```diff
@@ -117,3 +118,11 @@
 
 # sequence to be run on physics events (includes filters, reco and DQM)
 dtDQMPhysSequence = cms.Sequence(dtScalerInfoMonitor + gtDigis + reco + dtDQMTask + dtDQMTest)
+
+import EventFilter.DTRawToDigi.dturosunpacker_cfi
+_dturosunpacker = EventFilter.DTRawToDigi.dturosunpacker_cfi.dturosunpacker.clone()
+from Configuration.Eras.Modifier_run2_DT_2018_cff import run2_DT_2018
```

Comment:
era based modifications must be included in the python file that defines the module (and not propagated into the many other configurations that import it)